packages/field/src/react/__stories__/select-field.story.tsx

```diff
@@ -0,0 +1,115 @@
+import { CaretDownIcon } from '@pluralsight/ps-design-system-icon'
```

Comment:
Would be cool to see a Dropdown replacement using Dropdown.Menu or ActionMenu. Nbd, though.  Would just show that we've already replaced it.

Reply:
agreed. this is just an example of getting the functionality and a11y right. i assume we'll followup with an official `DropdownField` or `SelectField` component that would use an `ActionMenu`